### ISSUE TITLE
RCall not working after R update

### ISSUE THREAD

**Issue by pgurazada** [JuliaInterop/RCall.jl#239]
I updated my R version to 3.4.4 and cannot get RCall to work. The error is as follows:

    ERROR: LoadError: LoadError: Could not load library . Try adding  to the "PATH" environmental variable and restarting Julia.

I note that when I run the following 

    julia> ENV["R_HOME"]
    "C:\\Program Files\\R\\R-3.4.4\\bin\\x64"

I am assuming that it means the R installation is getting detected?

Further to this, I followed the instructions on the README page. 

    julia> ENV["R_HOME"] = "*"
    "*"

    julia> Pkg.build("RCall")
    INFO: Building SpecialFunctions

    julia> using RCall

The error now is:

    ERROR: LoadError: LoadError: Could not locate R installation. Try setting "R_HOME" environmental variable.

Following this I set the `R_HOME` variable.

    julia> ENV["R_HOME"] = "C:\\Program Files\\R\\R-3.4.4\\bin"

    julia> Pkg.build("RCall")
    INFO: Building SpecialFunctions

    julia> using RCall

Now the error is:

    ERROR: LoadError: LoadError: Could not load library . Try adding  to the "PATH" environmental variable and restarting Julia.

I seem to be going around in circles. Kindly suggest a solution.
![image](https://user-images.githubusercontent.com/35609736/38600451-2ffe9f7c-3d75-11e8-8543-98b375d2060b.png)

**Comment by randy3k** [comment]
How about

```
ENV["R_HOME"] = "C:\\Program Files\\R\\R-3.4.4\\bin\\x64"
Pkg.build("RCall")1
```

**Comment by pgurazada** [comment]
Sorry, I tried this too but the issue still persists. I tried out the path of R.exe from the i386 folder too, but it still does not load. 

![image](https://user-images.githubusercontent.com/35609736/38614194-4fd575a0-3d9d-11e8-987b-9e62dede56ce.png)

**Comment by randy3k** [comment]
Could you make sure you have the latest RCall?

**Comment by pgurazada** [comment]
You are right. JuliaPro is restricting the update of RCall beyond 0.8.1. Removing and re-installing the package installs 0.8.1 again. Possibly related to the restriction on update of DataFrames beyond 0.10.x? 

I tried things out on a fresh base Julia 0.6.2 install (i.e., not Julia Pro). I then tried installing `Pkg.add("RCall")`. But the error is that install did not work (see screenshot below). 

![image](https://user-images.githubusercontent.com/35609736/38662198-960800e0-3e44-11e8-84f1-0ef4efe54052.png)

Apologies for the long list of error screenshots, I am just trying to figure out where I am getting things wrong.

**Comment by pgurazada** [comment]
I am on the fresh Julia 0.6.2 install. I ran the following commands.

    julia> ENV["R_HOME"]
    "C:\\Program Files\\R\\R-3.4.4\\bin\\x64"

    julia> ENV["R_HOME"] = "C\\Program\ Files\\R\\R-3.4.4"
    "C\\Program Files\\R\\R-3.4.4" 

    julia> Pkg.build("RCall")
    INFO: Building SpecialFunctions
    INFO: Building CodecZlib
    INFO: Building Conda
    INFO: Building RCall
    INFO: Installing R via Conda.  To use a different R installation, set the "R_HOME" environment variable 
    and re-run Pkg.build("RCall").
    Warning: 'r' already in 'channels' list, moving to the top
    Solving environment: done

    # All requested packages already installed.

    ====================================================[ ERROR: RCall 
    ]====================================================

    LoadError: chmod: no such file or directory (ENOENT)
    while loading C:\Users\kimmcodxb\.julia\v0.6\RCall\deps\build.jl, in expression starting on line 53

Basically the same error as before. 😞

**Comment by pgurazada** [comment]
The Package install with the above `R_HOME` selection went fine, but post reloading Julia itself crashes. 😞 

![image](https://user-images.githubusercontent.com/35609736/38667406-6511d2fa-3e52-11e8-84c0-aa3cbbae292d.png)

**Comment by randy3k** [comment]
Did you try `Pkg.build("RCall")`?

**Comment by pgurazada** [comment]
I did a uninstall-reinstall and was able to build things correctly. The issue remains with the version of `RCall (0.8.1)` which ships with JuliaPro.

**Comment by randy3k** [comment]
Check https://github.com/JuliaInterop/RCall.jl/issues/225

**Comment by pgurazada** [comment]
Thank you! Yes, this issue has been bugging me since the ecosystem leaped beyond JuliaPro packages :) I need to be moving beyond JuliaPro for the moment till the next update. Thank you for your patience.
